WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -156,6 +168,39 @@ private extension CartView {
             cart: posModel.cart,
             orderStage: posModel.orderStage)
     }
+
+    func updateItemImageVisibility(dynamicTypeSize: DynamicTypeSize? = nil, cartListWidth: CGFloat) {
+        let newVisibility = cartListWidth >= minimumWidthToShowItemImages(with: dynamicTypeSize ?? self.dynamicTypeSize)
+        guard newVisibility != shouldShowItemImages else {
+            return
+        }
+        shouldShowItemImages = newVisibility
+    }
+
+    func minimumWidthToShowItemImages(with dynamicTypeSize: DynamicTypeSize) -> CGFloat {
```

Comment:
> Note that I chose to do the calculations on the cart, not each image. It just felt bad for performance to have lots of views using geometry readers to calculate the exact same answer... but it would be nicer from a code-readability perspective to have it on the item cards.

Theoretically, we could pass values of the geometry reader to the item view, and then it could decide whether it's shown or not. Of course, this way, responsibilities would be split between two views.

Reply:
Yeah, if we ever want it to be different between different types of cart item then we could do that, but it doesn't seem worth it just now